fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeJobV2.java

```diff
@@ -597,6 +599,16 @@ protected void runWaitingTxnJob() throws AlterCancelException {
                                                     new Field(col.getName(), col.getType(), tableName, null))
                                             .collect(Collectors.toList())));
 
+                            if (ConnectContext.get() == null) {
```

Comment:
shouldn't be null?

Reply:
This code path is only used for adding generated column. And if we analyze the expression of the generated column, we may need the ConnectContext to do it.